src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -834,6 +905,53 @@ private void replaceEC2Servers() {
         return updatedInstances;
     }
 
+    /**
+     * Attempts to terminate the provided instances. If the instances failed to terminate properly, the deploy job is
+     * failed. Returns true if the instances terminated successfully. Retruns false if instance termination encountered
```

Comment:
I think I added a commit to your other PR to fix this retruns typo. Not sure where it went...